### ISSUE TITLE
Line numbers being included with code listings starting with Sphinx 4.0.0

### ISSUE THREAD

**Issue by pvcraven** [readthedocs/sphinx_rtd_theme#1168]
### Problem

Starting with Sphinx 4.0.0 copying a block of code with `:linenos:` includes the line numbers along with the code when copied.

**To Reproduce**
Use Sphinx 4.0.0 or later. Create an RST that references an external file, something like:
```
.. literalinclude:: array_backed_grid.py
    :caption: array_backed_grid.py
    :linenos:
```

**Expected behavior**
The ability to copy code without line numbers. This worked fine with Sphinx 3.5.7 and before.Example:

<img width="393" alt="s2" src="https://user-images.githubusercontent.com/1463116/122603680-55237600-d03a-11eb-95d7-26fbbd0da906.png">

Starting with 4.0.0 we get:

<img width="426" alt="s1" src="https://user-images.githubusercontent.com/1463116/122603681-55237600-d03a-11eb-992d-81c5f1c63c20.png">

Sphinx copy button extension includes line numbers as well.

According to @mgeier the theme might need to be updated to support the new Sphinx 4.0.0 setup? See comments here:
https://github.com/sphinx-doc/sphinx/issues/9363

**Environment info**
- OS: Windows 10
- Python version: 3.9.5
- Sphinx version: 4.0.0 to 4.0.2

**Comment by Blendify** [comment]
This will be fixed by https://github.com/readthedocs/sphinx_rtd_theme/pull/1123

**Comment by Blendify** [comment]
Fixed by 380647d1596cec8bda15dc374aba270f884e6de8
